### PR TITLE
Update Zinit description + fix some related typos

### PR DESCRIPTION
**Pull request by psprint**
# Description

I've rebuilt the Zinit description and extended it – it now is enumeration oriented and a little longer than before. Is the length OK?

# Type of changes

- [ ] A link to an external resource like a blog post
- [ ] Add/remove a tab completion
- [ ] Add/remove a plugin
- [ ] Add/remove a theme
- [x] Text cleanups/typo fixes

# Copyright Assignment

- [x] This document is covered by the [BSD License](https://github.com/unixorn/awesome-zsh-plugins/blob/master/LICENSE), and I agree to contribute this PR under the terms of the license.

# Checklist:

- [x] I have confirmed that the link(s) in my PR are valid.
- [x] My entries are single lines and are in the appropriate (plugins, themes or completions) section, and in alphabetical order in their section.
- [x] I have read the **CONTRIBUTING** document.
- [x] All new and existing tests passed.
- [x] Any added completions have a license file in their repository.
- [x] Any added plugins have a license file in their repository.
- [x] Any added themes have a screen shot and a license file in their repository.
- [ ] I have stripped leading and trailing **zsh-** or **oh-my-zsh-** substrings from the link name. This makes it easier to find plugins/themes/completions by name since there aren't big clusters in the **O** and **Z** sections of the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,9 +180,29 @@ To do this, `zgen` generates a static `init.zsh` file which does nothing but sou
...
-[declare-zsh](https://github.com/zdharma/declare-zsh) to edit your Zinit
-commands in your `~.zshrc` and also use [zinit-console](https://github.com/zinit-zsh/zinit-console) to view and change the state of the ZSH session and to delete the plugins and snippets from the disk.
+**Zinit** is an innovative and currently the
+[fastest](https://github.com/zdharma/pm-perf-test) plugin manager with support
+for:
+
+- [Turbo
+mode](http://zdharma.org/zinit/wiki/INTRODUCTION/#turbo_mode_zsh_62_53) –
+**80%** faster Zsh startup! for example: instead of 200 ms, it'll be 40 ms!,
+- reports (from the plugin loads – plugins are no longer black boxes!),
```
[the remark this answers is not in the document]
Comment (psprint) [reply README.md]: I don't see any problem – the list *is* surrounded by blank lines.

[the remark this answers is not in the document]
Comment (unixorn) [reply README.md]: Some of the codeclimate complaints are bogus in this repo

File: README.md [hunk 2]
```diff
@@ -180,9 +180,29 @@ To do this, `zgen` generates a static `init.zsh` file which does nothing but sou
 
 ### [Zinit](https://github.com/zdharma/zinit)
 
-**Zinit** is an innovative plugin manager with [semigraphical UI](https://github.com/zdharma/zplugin-crasis), [Turbo Mode](http://zdharma.org/zinit/wiki/INTRODUCTION/#turbo_mode_zsh_62_53), a clean fpath, reports, completion management and [services](https://github.com/zservices) support. You can use [crasis](https://github.com/zdharma/zplugin-crasis) or
-[declare-zsh](https://github.com/zdharma/declare-zsh) to edit your Zinit
-commands in your `~.zshrc` and also use [zinit-console](https://github.com/zinit-zsh/zinit-console) to view and change the state of the ZSH session and to delete the plugins and snippets from the disk.
+**Zinit** is an innovative and currently the
+[fastest](https://github.com/zdharma/pm-perf-test) plugin manager with support
```
Comment (unixorn) [inline README.md]: please don't embed line breaks inside paragraphs - I prefer to let the markdown renderer handle them, and it makes it easier when editing the file to move paragraphs around when they're single lines.

Comment (psprint) [reply README.md]: I was editing in Vim and the long lines caused some problems, so I thought that I'll have them broke. It is withdrawn now!

**Pull request merged**